### PR TITLE
Adding CORS filter to the api

### PR DESCRIPTION
**Pull request by hselvakumar**
Jira Ticket : https://companieshouse.atlassian.net/browse/SEC-50

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by NeilShadrach** [comment]
No tests?

**Comment by hselvakumar** [comment]
> No tests?

sorry @NeilShadrach forgot to commit it and just committed now

File: src/main/java/uk/gov/companieshouse/charges/data/config/WebSecurityConfig.java [hunk 1]
```diff
@@ -27,11 +32,13 @@ public SecurityFilterChain filterChain(HttpSecurity http, Logger logger) throws
                 .csrf(AbstractHttpConfigurer::disable)
                 .formLogin(AbstractHttpConfigurer::disable)
                 .logout(AbstractHttpConfigurer::disable)
+                .cors(AbstractHttpConfigurer::disable)  // Added CORS disable
```
Comment (NeilShadrach) [inline src/main/java/uk/gov/companieshouse/charges/data/config/WebSecurityConfig.java:35]: I don't think the comments here and L38 add anything

**Review by NeilShadrach — DISMISSED** [review]

File: src/main/java/uk/gov/companieshouse/charges/data/config/WebSecurityConfig.java (hunk 1 above)
Comment (hselvakumar) [reply src/main/java/uk/gov/companieshouse/charges/data/config/WebSecurityConfig.java:35]: removed 👍

**Review by NeilShadrach — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**